London_10-Saqib_Javed_JavaScript-Core-2-Coursework-Week2

saqibjvd opened the pull request:
<!--

The title for your pull request should be made in this format

CITY CLASS_NO - FIRST_NAME LAST_NAME - MODULE - WEEK_NO

For example,

London Class 7 - Chris Owen - HTML/CSS - Week 1

Please complete the details below this message

-->

**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

saqibjvd committed:
1. writers

saqibjvd committed:
Update 2-people-i-know.js

saqibjvd committed:
reading list

saqibjvd committed:
2-people-i-know.js

few upper case letter made code not working. fixed it

saqibjvd committed:
budget

kwebster77 commented:
Hi @saqibjvd -- It looks like the tests for `3-reading-list `failed. Want to give it another try? 😉 If you have any Qs, reach out to our buddy group!

saqibjvd commented:
Hi.I have made changes you recommended could you please check them. Thanks

saqibjvd committed:
reading-list

updated my code so make sure code runing and getting same result as expected. works fine
